_extras/snippets_library/UCL_sge/14/load_python.snip

```diff
@@ -1,2 +1,2 @@
-[yourUsername@cirrus-login0 ~]$ module load anaconda/python3
-[yourUsername@cirrus-login0 ~]$ which python3
+[cceagil@login12 files]$ module load python
+[cceagil@login12 files]$ which python3
```

Comment:
Instead of using your username, use the general `yourUsername` as in the other example.

Reply:
In fact, maybe using `{{ site.host_prompt }} ` works...

Reply:
Yes, I need to change the places where I hardcoded my prompt. I should use ``{{ site.host_prompt }}`` everywhere